### ISSUE TITLE
iuser.key doesn't get created and user accounts aren't created on nodes

### ISSUE THREAD

**Issue by sstine** [JohnOmernik/zetago#24]
I completed Zeta prep walkthrough but it seems to have failed silently. I am able to ssh into the agent nodes from the master node by using the command `ssh -i ~/.ssh/zeta.pem ubuntu@node-ip` but I cannot SSH in as `zetaadm` or `mapr` as the accounts were not created. The private key was never copied to `./conf/iuser.key` on the master node.

```bash
$ ssh -i ~/.ssh/zeta.pem zetaadm@54.202.106.68
Permission denied (publickey).
```

```bash
$ tree conf
conf
├── base.conf
└── prep.conf
```

Log of the setup walkthrough:

```bash
-----------------------------------
This process will ask a series of questions in order to create a prep_conf.conf script for gettings nodes ready for Zeta
The configuration will be located: ./conf/prep.conf
-----------------------------------

Initial User for Connection to Zeta Cluster
We need access to the initial nodes on the Zeta cluster. We do this through a username that has passwordless sudo and a key without a password
Once the zeta cluster is created, this user can be deprivileged.

We do try to detect this, through a list of users, you can provide a space separated list of users, or take ours as default

For example, if this was AWS, and we were using ubuntu based nodes, I'd answer 'ubuntu' for the Initial users
If I were using ubuntu and centos I'd provide 'ubuntu centos' for the Intial users

Initial user list to connect to zeta nodes: ubuntu

We now need the key for the initial connection, this is typically location in /home/ubuntu/.ssh/id_rsa, but doesn't have to be. Please specify that here

Private key for initial user on zeta nodes: /home/ubuntu/.ssh/zeta.pem


As part of this process, we will generate a key for the IUSER user
It will be stored at ./conf/iuser.key and its public key will be stored at ./conf/iuser.key.pub
These values will be saved in the conf for use as well

-----------------------------------
The UID for the IUSER user will be 2500 and the UID for the mapr user will be 2000
These values are stored in the conf

-----------------------------------

Please enter password for zetaadm:
Please re-enter password for zetaadm:

Please enter password for mapr:
Please re-enter password for mapr:

-----------------------------------
Please enter the list of nodes we will be preparing for Zeta, the nodes should be space separated

Enter list of space separated nodes for use in Zeta: 54.202.106.68 54.200.53.73 54.186.17.222 54.213.171.63

In setting up Zeta, we like to select an 'Initial' node that can be used as the node that all the scripts run from. It should be an agent node

Enter initial node for Zeta install: 54.202.106.68


-----------------------------------
As we are prepping the nodes, it's helpful to have a list of interfaces to check for the internal IP Address of the nodes.
Basically provide a space separated list, and the first interface in that list that returns an IP will be the lucky winner

Interface to check list: eth0 em1 eno1 enp2s0 enp3s0 ens192
INFO   Prep Conf file written to ./conf/prep.conf
Commands to use with prep:

./zeta prep userprep -n=NODE     # Prep user on a specific node
./zeta prep systemprep -n=NODE   # Prep system on a specific node
./zeta prep install -n=NODE1     # Run both the userprep and systemprep on NODE1
./zeta prep install -a           # Prep User and system on all nodes but do so one at a time
./zeta prep install -a -u        # Prep all nodes in prep.conf in parallel


To lock or unlock the prep config:
./zeta prep -l


To log into the initial node, please use:

ssh -i ./conf/iuser.key zetaadm@54.202.106.68
```

**Comment by JohnOmernik** [comment]
After you run ./zeta prep it only creates the configuration, to actually prep the nodes, you need to run ./zeta prep install -a -u

Did you run that?
